src/sentry/static/sentry/app/views/organizationDiscover/styles.jsx

```diff
@@ -50,6 +50,11 @@ export const BodyContent = styled(Flex)`
   padding: ${space(1.5)} 32px 32px 32px;
 `;
 
+export const Loading = styled(Flex)`
+  flex: 1;
+  height: calc(100vh - ${FOOTER_HEIGHT}px);
```

Comment:
nice calc-ing! I can't think of any problems this would cause. 100vh will factor in things like collapsing url bars and bottom-bars on mobile devices but I think that is probably the desired behavior here.

Reply:
Hmm, I actually don't think page height is working properly on mobile devices for any of discover since the footer appears to be hidden <768 pixels so all the calculations are wrong. This fixes the loader problem for desktop only.

Reply:
How do you generally deal with this in styled components? Media queries, or sth else?

Reply:
I think Media Queries are a good idea. They seem too useful to leave off the table. Styled components accept nested MQ syntax:

```
const Container = styled("div")`
  height: 700px;

  @media (min-height: 700px) {
    height: 100vh;
  }
`;
```

or depending on the situation you could maybe get away with just `max-height: 700px` or something;

some atomic frameworks have built in responsiveness but I don't think grid-emotion does.